### ISSUE TITLE
Lesson Feedback: Basics - Lesson 3 (Read nested data using links)

### ISSUE THREAD

**Issue by ajhurliman** [ProtoSchool/protoschool.github.io#210]
Have a question or suggestion regarding a specific ProtoSchool lesson? Please use this template to share it!

**URL of the lesson that's confusing:**
eg https://proto.school/#/blog/01

**What's confusing about this lesson?**
This question in particular:
```
/* globals ipfs */

const run = async () => {
  let cid = await ipfs.dag.put({ test: 1 })
  let cid2 = await ipfs.dag.put({ bar: cid })
  // your code goes here
  let val = await ipfs.dag.get(cid2, '/bar/test');
  let returnVal = val.value
  return returnVal;
}

return run
```
It's tempting to try to get the value off of the promise like so: `await ipfs.dag.get(cid2, '/bar/test').value;` but obviously the promise doesn't have a `value` property on it until it resolves. Since there's no STDOUT you can't see what it is either.

**What additional context could we provide to help you succeed?**

**What other feedback would you like to share about ProtoSchool?**

**Comment by terichadbourne** [comment]
Thanks for pointing that out. I think we put a clue in another lesson about wrapping the entire await statement in parens but we missed it here. 

I’m afraid I’m not familiar with STDOUT. Can you tell me more about what feature you would have hoped to see here?

**Comment by terichadbourne** [comment]
resolved by #234
